information/src/main/java/org/polypheny/db/information/InformationDuration.java

```diff
@@ -62,6 +61,15 @@ public Duration get( final String name ) {
     }
 
 
+    public long getNanoDuration( final String name ) {
+        Duration child = this.get( name );
+        if ( child != null ) {
+            return child.duration;
+        }
+        return 0;
```

Comment:
It would be better to throw an Exception. Such "default" values are always dangerous....